servers/rendering/rasterizer_rd/rasterizer_effects_rd.cpp

```diff
@@ -1356,6 +1402,22 @@ RasterizerEffectsRD::RasterizerEffectsRD() {
 		ssr_scale.pipeline = RD::get_singleton()->compute_pipeline_create(ssr_scale.shader.version_get_shader(ssr_scale.shader_version, 0));
 	}
 
+	{
+		Vector<String> sss_modes;
+		sss_modes.push_back("\n#define USE_11_SAMPLES\n");
+		sss_modes.push_back("\n#define USE_17_SAMPLES\n");
+		sss_modes.push_back("\n#define USE_25_SAMPLES\n");
+		//sss_modes.push_back("\n#define VERTICAL_PASS\n");
+
+		sss.shader.initialize(sss_modes);
+
+		sss.shader_version = sss.shader.version_create();
```

Comment:
Shader is created, but never freed. Should be freed in the destruction function. Should also add code to free the SSR shaders while you are at it.